DeadCrumbs/app/src/main/java/dead/crumbs/ui/MapsViewModel.kt

```diff
@@ -71,14 +70,14 @@ class MapsViewModel (private val mapsRepository: MapsRepository) : ViewModel(){
                 }
             //adding the meMarker to the map
             val marker = map.addMarker(newMarker( loc = LatLng(loc!!.latitude, loc.longitude),
-                name = "Me", icon = R.mipmap.my_picture))
+                name = MainActivity.my_username, icon = R.mipmap.arrow))
```

Comment:
I think we should talk about the arrow. It seems the sprite for it is a bit wonky